R/bmi.R

```diff
@@ -1,107 +1,123 @@
-#' @title ifelse2
-#' 
+#' @title if_else2
+#'
 #' @description Custom ifelse function that evaluates missing (NA) values
-#' 
-#' @details If the logical argument (x) compares to a value that is `NA`, it is set to `FALSE`  
-#' 
+#'
+#' @details If the logical argument (x) compares to a value that is `NA`, it is
```

Comment:
Can you add a comment as to why we need this as opposed to just using `ifelse` in the description?